packages/gatsby-theme-styleguide/src/gatsby-components/LstVersionLink/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
```

Comment:
I think a better name would be `designerKitLink`

you can also follow the pattern : 

index.js : 
```js
export { default } from './designerKitLink'
```

designerKitLink : 
```js
// your code
```

That way when developers have multiple component opened in their editors, they don't have only 'index' tabs opened.